aplib.net-demo/Assets/Scripts/WFC/GridPlacer.cs

```diff
@@ -126,9 +155,31 @@ public void MakeScene()
 
             PlaceGrid();
 
+            Cell randomPlayerSpawn = Grid.GetRandomFilledCell();
+
+            SetRandomPLayerSpawn(randomPlayerSpawn);
+
             JoinConnectedComponentsWithTeleporters();
 
-            PlacePlayer();
+            PlaceDoorsBetweenConnectedComponents(randomPlayerSpawn);
+        }
+
+        /// <summary>
+        /// Waits before making the scene.
+        /// </summary>
+        /// <param name="waitTime">The time to wait before making the scene.</param>
+        public void WaitBeforeMakeScene(float waitTime = 0.01f) => StartCoroutine(Wait(waitTime));
+
+        /// <summary>
+        /// Waits for a certain amount of time.
+        /// </summary>
+        /// <param name="waitTime">The time to wait before making the scene.</param>
+        /// <returns>An <see cref="IEnumerator" /> that can be used to wait for a certain amount of time.</returns>
+        private IEnumerator Wait(float waitTime)
```

Comment:
Since this method also calls the MakeScene method, I would rename it. It does not just wait.
Something like `WaitThenMakeScene`.